src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,48 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string,
+  description?: string,
+  imgUrl: string,
+  imdbUrl: string,
+};
+
+export const MovieCard: React.FC<Props> = (props) => (
+  <>
+    <div className="card">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            src={props.imgUrl}
```

Comment:
At start you can spread `props` like `const { title, description, imgUrl, imbdUrl } = props;`  and use it instead `props.imgUrl`